nrf_security/configs/nrf-config.h.template

```diff
@@ -3721,8 +3721,8 @@ it is (2^48 - 1), our restriction is :  (int - 0xFFFF - 0xF).*/
  * Allow user to override any previous default.
  *
  */
-#if defined(MBEDTLS_USER_CONFIG_FILE)
-#include MBEDTLS_USER_CONFIG_FILE
+#if defined(CONFIG_MBEDTLS_USER_CONFIG_FILE)
```

Comment:
We don't add kconfig in the template
The mechanism required is the addition of something that converts CONFIG_MBEDTLS_XXXX into MBEDTLS_XXX

I will contact you and show what is needed

Reply:
Adding comment here as well for posterity:
We convert the configuration from **CONFIG_MBEDTLS_XXXX** in **nrf_security/cmake/kconfig_mbedtls_configure.cmake** with the macro **kconfig_mbedtls_config** which will strip CONFIG_ from the configuration (and make it the same format as mbed TLS expects):

It is necessary to make **#cmakedefine MBEDTLS_XXXX** inside nrf-config.h.template that will be replaced either with a defined version or an unset version (when the config is not to be present)

Reply:
I fixed it and it's good thing that I did. There is another problem about it - the header file is not seen for some sources
```
/home/pdunaj/work/ncs/mbedtls/library/ecjpake.c:25:
modules/nrfxlib/nrf_security/include/nrf-config.h:3724:34: fatal error: config-tls-user.h: No such file or directory
 3724 | #define MBEDTLS_USER_CONFIG_FILE "config-tls-user.h"
```

The problem is the include paths added by nrf_security are very limitted and library does not see directory with zephyr generated files (where we put our config file). I also don't think it will be possible to add header directory by the application as it is already pass the boilerplate :/

Seems I will have to go and add every missing option (I planned to do it anyway but it seems I will have no other choice).

Reply:
I have added the missing config files but left the user config option too - just in case it can be useful later on.